apps/src/blockly/TESTING.md

```diff
@@ -4,13 +4,13 @@ Whenever the mainline `blockly` package version is being upgraded or updates are
 The following is a manual test checklist to follow and is not exhaustive. The estimated time to complete the following checklist is 15-20 minutes.
 
 # For All Mainline Blockly Labs
-Currently, the mainline Blockly labs are: Dance, Poetry, Music (Project Beats), Flappy, Bounce.
+Currently, the mainline Blockly labs are: Dance, Poetry, Music, Flappy, Bounce.
```

Comment:
Thanks for such a thorough clean up! This made me realize we should add Minecraft to this list and replace Poetry with Sprite Lab (its parent lab).